.gitignore

```diff
@@ -16,3 +16,5 @@ node_modules
 /postgres-data
 
 /app/styles/tailwind.css
+
+/.idea
```

Comment:
This should be in the global `.gitignore`
https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

```suggestion
```